python/orca/tutorial/NCF/pytorch_model.py

```diff
@@ -22,37 +22,46 @@
 
 
 class NCF(nn.Module):
-    def __init__(self, user_num, item_num, factor_num, num_layers,
-                 dropout, model, GMF_model=None, MLP_model=None):
+    def __init__(self, factor_num, num_layers,
+                 dropout, model, GMF_model=None, MLP_model=None,
+                 cat_feats_dim=0, numeric_feats_dim=0, num_embed_cat_feats=None):
```

Comment:
I think we only need these arguments?
- cat_feats_dims -> this can calculate num_cat_feats
- num_numeric_feats
- Probably the embedding_dim of cat feats?